.github/workflows/ci.yml

```diff
@@ -104,6 +106,9 @@ jobs:
     steps:
       - uses: actions/checkout@v4
 
+      - name: Setup sccache
+        uses: mozilla-actions/sccache-action@v0.0.4
```

Comment:
The latest version is 0.0.7, but I only see 0.0.4 listed here https://github.com/apache/infrastructure-actions/blob/main/approved_patterns.yml

Reply:
heres the original issue for 0.0.4

https://issues.apache.org/jira/browse/INFRA-25525?jql=text%20~%20%22sccache-action%22